### PR TITLE
Modification on the trigger build function, in order to be able to aCcept a more general payload (env variables). Created a function for triggering multiple builds at once.

### PR DESCRIPTION
**Pull request by PapadakisGeorge**

### CODE REVIEW

File: src/bitriseAPI/triggerMultipleBuilds.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+const {consoleRed, consoleGreen} = require('../utils/consoleColors');
+const {createPayload} = require("../utils/createPayload");
+const {triggerBuild} = require("./triggerBuild");
+
+
+/**
+ * @param builds A list with each builds' desired branch, workflow and environmental variables.
+ */
+const triggerBuildMultipleBuilds = async (builds) => {
```
Comment (kkmavr) [inline src/bitriseAPI/triggerMultipleBuilds.js]: Could you change that to `triggerMultipleBuilds?`

File: src/bitriseAPI/triggerStart.js [hunk 2]
```diff
@@ -29,57 +30,66 @@ const triggerStart = async (initialWorkflow='') => {
...
+    //         "base_repository_url": "https://github.com/camelotls/ie-native-app",
+    //         "environments": [
+    //             {
+    //                 "is_expand": true,
+    //                 "mapped_to": "SKIP_TESTS",
+    //                 "value": YES_OPTIONS.includes(SKIP_TESTS) ? "true" : "false"
+    //             },
+    //         ]
+    //     }
+    // }
```
Comment (kkmavr) [inline src/bitriseAPI/triggerStart.js]: If these are not needed, could you please remove them?

**Review by kkmavr — CHANGES_REQUESTED** [review]

File: src/bitriseAPI/triggerMultipleBuilds.js (hunk 1 above)
Comment (PapadakisGeorge) [reply src/bitriseAPI/triggerMultipleBuilds.js]: You are absolutely right, my mistake.

File: src/bitriseAPI/triggerStart.js (hunk 2 above)
Comment (PapadakisGeorge) [reply src/bitriseAPI/triggerStart.js]: I might do that, but shouldn't would stash code for when the difficult times come?

**Pull request merged**